app/helpers/patient_query_helper.rb

```diff
@@ -380,13 +380,10 @@ def advanced_filter(patients, filters, tz_offset)
           #     [{"label": "USA", "value": "1"}, {"label": "USA, State 1", "value": "2"}],
           #   would map to
           #     Jurisdiction.where([1, 2])
-          filter_jurisdictions = Jurisdiction.where(id: filter[:value].map(&:values).map(&:second).map(&:to_i))
-
-          # Get subset of jurisdictions
-          subjurisdictions = filter_jurisdictions.map(&:subtree_ids).flatten.uniq
+          filter_jurisdiction_ids = Jurisdiction.where(id: filter[:value].pluck(:value)).pluck(:id)
 
           # Get patients where jurisdiction is any of the jurisdictions or subjurisdictions of the filter
```

Comment:
You can remove `or subjurisdictions` from this comment

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1216/commits/792b83a5c19cbf4c5a5395191a35ffb73d649b73